plugins/rest_connector/api/tasks.py

```diff
@@ -79,9 +82,27 @@ def perform_request(self, connector_id: str, db_id: int) -> None:
         timeout=20,  # TODO allow different timeouts?
     )
 
-    # FIXME response handling...
+    template_context: Dict[str, Any] = request_variables | {
+        "request": response.request,
+        "response": response.json(),
```

Comment:
TODO: Do not use `json()` if not requested by the template. (this requires a proxy object or just passing the request).